.github/workflows/build-test-measure.yml

```diff
@@ -360,7 +360,7 @@ jobs:
       WP_CORE_DIR: /tmp/wordpress
     services:
       mysql:
-        image: mariadb:latest
+        image: mariadb:10
```

Comment:
> **Note**

The most recent MariaDB images appear to have a problem that prevents DB services from starting up. Once this is rectified in a future release, this change may be undone. In the meantime, `mearidb:10` will work for us going forward.

Reply:
@thelovekesh Here's a more future-proof workaround: https://github.com/swissspidy/preferred-languages/commit/ce3f0d532283c8f4b4908186ec077f34076dc367

Reply:
Thanks @swissspidy. This will be updated as a part of #7560.